packages/sample-project/cannonfile.toml

```diff
@@ -49,17 +51,20 @@ depends = [
 [contract.Router]
 artifact = "contracts/routers/chain-<%= chainId %>/Router.sol:Router"
 abiOf = [
-  "OwnerModule",
-  "UpgradeModule",
+  "core_router.CoreModule",
   "InitializableModule",
   "SettingsModule",
   "SomeModule",
   "AnotherModule"
 ]
 depends = ["run.generate_router"]
 
-[contract.Proxy]
-artifact = "contracts/Proxy.sol:Proxy"
+[invoke.upgrade_proxy]
```

Comment:
i agree with the change, but unfortunately for v3 synthetix-main we have already committed to the deployment steps that we have for the current router, so we can't do an upgrade without completely resetting the proxy. for future projects, this is definitely the approach, but I just want to make sure we are not making any changes to that?